src/theme/dojo/outlined-button.m.css

```diff
@@ -0,0 +1,63 @@
+@import './variables.css';
+
+.root {
+	background-color: var(--color-background);
+	border: var(--border-width) solid var(--color-highlight);
+	color: var(--color-highlight);
+	cursor: pointer;
+	display: inline-block;
+	font-size: var(--font-size-base);
+	font-weight: bold;
+	line-height: var(--line-height-base);
+	min-width: calc(var(--grid-base) * 20);
+	padding: var(--spacing-regular);
+	transition: box-shadow var(--transition-duration) var(--transition-easing);
+	margin: 0;
+}
+
+.root:hover {
+	background-color: var(--color-highlight);
+	color: var(--color-text-inverted);
+	outline: none;
+}
+
+.root:hover .addon,
+.root:focus .addon {
+	border-left-color: var(--color-highlight);
+}
+
+.pressed {
+	background-color: var(--color-highlight);
+	border-color: var(--color-highlight);
+	color: var(--color-text-inverted);
+}
+
+.pressed .addon {
+	border-left-color: var(--color-highlight-border);
+}
+
+.popup {
```

Comment:
popup and addon are no longer used, can you remove them from `button.m.css` as well please?

Reply:
Removed :)